lib/mozart_fetcher/component.ex

```diff
@@ -34,7 +34,11 @@ defmodule MozartFetcher.Component do
   end
 
   defp get(config) do
-    LocalCache.get_or_store(config.endpoint, fn -> HTTPClient.get(config.endpoint) end)
+    headers = config.headers || %{}
+
+    LocalCache.get_or_store(config.endpoint, fn ->
+      HTTPClient.get(config.endpoint, Map.to_list(headers))
```

Comment:
Will this work?
I was thinking that we needed a list of Tuples but this would convert it to a list of keyword values.
```elixir
iex(1)> a = %{foo: "bar"}
%{foo: "bar"}
iex(2)> Map.to_list(a)
[foo: "bar"]
iex(3)> b = %{foo: "bar", baz:  "far"}
%{foo: "bar", baz: "far"}
iex(4)> Map.to_list(a)
[foo: "bar"]
iex(5)> Map.to_list(b)
[foo: "bar", baz: "far"]
```

Reply:
Even if you have key that are string is returning list of tuples
```elixir
iex(2)> Map.to_list(%{"a" => 42})
[{"a", 42}]
```

Reply:
Ahh okay nice, wasn't quite sure so thought I'd double check :)